string device_path
string[] channels
int32 samples_count
---
bool success
string message
std_msgs/MultiArrayLayout layout
